tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -182,7 +182,9 @@ def update_root(self, data: bytes) -> None:
         self._trusted_set["root"] = new_root
         logger.info("Updated root v%d", new_root.signed.version)
 
-    def update_timestamp(self, data: bytes) -> None:
+        return new_root
+
+    def update_timestamp(self, data: bytes) -> Metadata:
```

Comment:
``` suggestion
    def update_timestamp(self, data: bytes) -> Metadata[Timestamp]:
```
This will give hint to our IDEs that the return type is not just Metadata, but Timestamp.
See an example:
https://github.com/theupdateframework/python-tuf/blob/ec285a448c5fa2ef35f765bd297b147d92e408b1/tuf/ngclient/_internal/trusted_metadata_set.py#L162